### PR TITLE
Refactor bitesize7

### PR DESCRIPTION
**Pull request by t-young31**

### CODE REVIEW

File: tdms/src/iterator.cpp [hunk 1]
```diff
@@ -5878,12 +5622,6 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[]) {
   free(Dzu);
   //  free(lower_boundary_update);
   free(Nt);
-  free(I0);
-  free(I1);
-  free(J0);
-  free(J1);
-  free(K0);
-  free(K1);
```
Comment (samcunliffe) [inline tdms/src/iterator.cpp:5886]: Automatically/implicitly in the InterfaceComponent destructor? I don't see the explicit `free` ing.

**Review by samcunliffe — APPROVED** [review]
Right. I think I've got all of that too. It all looks good... 

I really don't spot anything. Which is, itself, a little suspicious.

File: tdms/src/iterator.cpp (hunk 1 above)
Comment (t-young31) [reply tdms/src/iterator.cpp:5886]: yep – assumed we're all good with an implicit trivial destructor if the members are only an int and bool(??)

**Comment by t-young31** [comment]
> I really don't spot anything. Which is, itself, a little suspicious.

_very_ suspicious..

**Pull request merged**